tests/integration/test_vulnerability_detector/test_providers/test_multiple_provider_feeds.py

```diff
@@ -111,13 +111,16 @@ def check_log_events(metadata):
     if 'Debian Buster' in metadata['provider_name']:
         evm.check_fetching_feed_log(metadata['json_feed_path'])
         evm.check_refresh_feed_log(metadata['provider_name'])
-        evm.check_provider_database_update_finish_log(metadata['provider_name'])
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=None,
+                                                      provider_name=metadata['provider_name'])
     elif 'Red Hat Enterprise Linux 8' in metadata['provider_name']:
         evm.check_refresh_feed_log(metadata['provider_name'])
-        evm.check_provider_database_update_finish_log(metadata['provider_name'])
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=None,
```

Comment:
```suggestion
        evm.check_provider_database_update_finish_log(
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800